src/TLuaInterpreter.cpp

```diff
@@ -1962,16 +1924,14 @@ int TLuaInterpreter::getLineCount(lua_State* L)
 {
     Host& host = getHostFromLua(L);
     if (lua_isstring(L, 1)) {
-        std::string window = lua_tostring(L, 1);
-        QString _window = window.c_str();
-        lua_pushnumber(L, mudlet::self()->getLastLineNumber(&host, _window) + 1);
+        QString window = lua_tostring(L, 1);
```

Comment:
Need to convert it from UTF-8 - should be:
```cpp
QString window = QString::fromUtf8(lua_tostring(L, 1));
```
or even more concisely (and making it `const` to help the compiler in opimisations):
```cpp
const QString window{QString::fromUtf8(lua_tostring(L, 1))};
```

Reply:
This may be a stupid question, but why would we need QString::fromUtf8 here? Or in fact anywhere?

The QString documentation clearly states that QString::fromUtf8 is implicit in constructors from `const char*` (which includes assignment AFAICT).